.github/workflows/nightly-te-test.yaml

```
@@ -25,7 +25,7 @@ permissions:
   packages: write # to upload container
 
 env:
-  DEFAULT_JAX_TE_IMAGE: 'ghcr.io/nvidia/jax-te:latest'
+  DEFAULT_JAX_TE_IMAGE: 'ghcr.io/nvidia/jax:latest'
```

Same Q about upstream-pax
```suggestion
  DEFAULT_JAX_TE_IMAGE: 'ghcr.io/nvidia/upstream-pax:latest'
```

---

Fixed in 6974a3ad